app/_locales/en/messages.json

```diff
@@ -632,6 +632,9 @@
   "busy": {
     "message": "Busy"
   },
+  "buy": {
+    "message": "Buy"
+  },
   "buyAndSell": {
     "message": "Buy & Sell"
   },
```

Comment:
Could you elaborate on why this needed to be added? It may be that the tests were failing because it wasn't up to date with `develop`

Reply:
@georgewrmarshall While seemingly unrelated, there's a test that is involved in the change of the network display component. If you'd like to see what I'm referring to I removed the change and pushed. The failing test is shown below in "
ci/circleci: test-unit-jest-main". The change below is also related to the same test failing.

Reply:
This issue may have been introduced by another PR which seems to have been resolved so I think we can revert these changes and try again with the latest changes from `develop`

Reply:
Yea looks like it. I can see someone else snuck this identical code in there recently so it's resolved